autoload/youcompleteme.vim

```diff
@@ -505,6 +508,8 @@ function! s:OnInsertEnter()
     return
   endif
 
+  call s:SetOmnicompleteFunc()
```

Comment:
This would set the omnifunc on every single InsertEnter event; what's the possible fallout from this?